src-java/blue-green/src/main/java/org/openkilda/bluegreen/ZkClient.java

```diff
@@ -86,7 +86,7 @@ public synchronized void initAndWaitConnection() {
                 .onRetry(e -> {
                     String message = format("Failed to init zk client, retrying... Attempt: %d", e.getAttemptCount());
                     if (e.getAttemptCount() <= 10) {
-                        log.info(message, e.getLastException());
+                        log.info(format("%s. Error: %s", message, e.getLastException().getMessage()));
```

Comment:
It might be better to use already existed "{}" placeholders for Slf4j here, instead of using String.format placeholders..

```suggestion
                        log.info("{}. Error: {}", message, e.getLastException().getMessage());
```